src/TacticsUtil.v.cppo

```diff
@@ -13,7 +13,7 @@ From Ltac2 Require Import Ltac2.
 From Ltac2 Require Import Message.
 From Ltac2 Require Import Constr.
 
-Ltac2 ltac1_congruence () := Ltac1.run (Ltac1.ref [ident:(Coq); ident:(Init); ident:(Prelude); ident:(congruence)]).
+Ltac2 ltac1_congruence () := Ltac1.run (Ltac1.ref [ident:(Stdlib); ident:(Init); ident:(Prelude); ident:(congruence)]).
```

Comment:
To make this backwards compatible you could use Env.path, eg
~~~coq
  let stdlib_or_coq :=
    match Env.path reference:(eq) with
    | [] => Control.throw Assertion_failure
    | v :: _ => v
    end
  in
  [stdlib_or_coq; ident:(Init); ident:(Prelude); ident:(congruence)]
~~~
(or maybe just doing `Ltac2 ltac1_congruence () := ltac1:(congruence).` would work)

Reply:
or use the cppo #if stuff